docs/src/modules/components/overview/charts/essentialCharts/PieChartDemo.tsx

```diff
@@ -3,6 +3,8 @@ import { pieArcLabelClasses, PieChart } from '@mui/x-charts/PieChart';
 import Typography from '@mui/material/Typography';
 import Stack from '@mui/material/Stack';
 import ChartDemoWrapper from '../ChartDemoWrapper';
+import { useDrawingArea } from '@mui/x-charts/hooks';
+import { styled } from '@mui/material/styles';
 
```

Comment:
@prakhargupta1 The convention is to group all package imports before the relative imports.

In that case, the import `from '@mui/...` should be before the one `from '../ChartDemoWrapper'`. Otherwise you break the lint rules.

Even better to group the per package (all `@mui/material` together, all `@mui/x-charts` together ...) to simplify the reading